### ISSUE TITLE
Shawi: order of elements in cit type example

### ISSUE THREAD

**Issue by rausch-supola** [acdh-oeaw/generic-dict-schema#5]
In cit type example often a usg element occurs before quote which causes errors at the moment. Analogous to other parts of the dictionary, would it be possible to put the usg type geographic after quote or after cit type translations? 

![image](https://github.com/user-attachments/assets/22efd1ad-7a7e-49ba-a78c-50417832ca7e)

@dasch124

**Comment by dasch124** [comment]
In today's meeting we have decided that `<quote>` should be the first element within `<cit>`, as it is also for the other `@type`s of `<cit>` elements. Of course, we can make an exception for examples, but I feel that it would be better to keep the encoding of the various types as close to each other as possible to not overcomplicate the schema. I would propose to move `<usg>` to the end, i.e. before `<fs>` so the `<quote>` and its translations are kept together. Would that be ok, @charlymo ?

**Comment by charlymo** [comment]
I agree to the principle. But, cit/quote only exists in examples, or am I overlooking something?
Oh, just found 69 sense/cit/quote. Die dürfte es eigentlich gar nicht mehr geben. Da müsste das Schematron doch eigentlich schreien?

Yes, usg after quote, before fs. I am not happy with ***@***.***=”geographic”] there anyway.

From: dasch124 ***@***.***>
Sent: Tuesday, July 30, 2024 5:52 PM
To: acdh-oeaw/generic-dict-schema ***@***.***>
Cc: Mörth, Karlheinz ***@***.***>; Mention ***@***.***>
Subject: [EXT] Re: [acdh-oeaw/generic-dict-schema] Shawi: order of elements in cit type example (Issue #5)

External Email: please use links and attachments from trusted sources only


In today's meeting we have decided that <quote> should be the first element within <cit>, as it is also for the other @types of <cit> elements. Of course, we can make an exception for examples, but I feel that it would be better to keep the encoding of the various types as close to each other as possible to not overcomplicate the schema. I would propose to move <usg> to the end, i.e. before <fs> so the <quote> and its translations are kept together. Would that be ok, @charlymo<https://github.com/charlymo> ?

**Comment by rausch-supola** [comment]
In our meeting last Tuesday we discussed that we should simply add a possibility for usg before quote and cit, which I added recently (https://github.com/acdh-oeaw/generic-dict-schema/commit/9ab52cce9c975e74c4f455a56b023625544aa0e8)

> we have decided that <quote> should be the first element within <cit>

So, there should be no usg before quote? 

> Die dürfte es eigentlich gar nicht mehr geben. Da müsste das Schematron doch eigentlich schreien?

Due to issue #6 the constraint rules are currently not applied. I will check this when I can create a compiled odd again (I have an error with odd2odd.xsl)
